loader/CMakeLists.txt

```diff
@@ -435,6 +435,13 @@ if(PKG_CONFIG_FOUND)
 endif()
 
 target_link_libraries(vulkan PRIVATE Vulkan::Headers)
-add_library(Vulkan::Vulkan ALIAS vulkan)
+add_library(Vulkan::Loader ALIAS vulkan)
 
-install(TARGETS vulkan)
+if (APPLE AND BUILD_STATIC_LOADER)
+    # When exporting a static library all linked libraries - private or not - need to be exported.
+    return()
+endif()
```

Comment:
Found the problem causing Apple builds to fail. Has nothing to do with Apple. It has to do with the rules when you export a static library.

Reply:
So the solution is to not export anything if we are building a static loader on macOS?

Reply:
yep since this is only needed for the chromium build